### PR TITLE
Specify Element.ALLOW_KEYBOARD_INPUT as an integer type

### PR DESCRIPTION
**Pull request by realityforge**

### CODE REVIEW

File: java/elemental2/dom/integer_entities.txt [hunk 1]
```diff
@@ -92,13 +92,16 @@ elemental2.dom.DOMTokenList.length
...
+elemental2.dom.Element.ALLOW_KEYBOARD_INPUT
 elemental2.dom.Element.childElementCount
 elemental2.dom.Element.clientHeight
 elemental2.dom.Element.clientLeft
 elemental2.dom.Element.clientTop
 elemental2.dom.Element.clientWidth
 elemental2.dom.Element.scrollHeight
 elemental2.dom.Element.scrollWidth
+elemental2.dom.Element.webkitRequestFullScreen.allowKeyboardInput
+elemental2.dom.Element.webkitRequestFullscreen.allowKeyboardInput
```
Comment (jDramaix) [inline java/elemental2/dom/integer_entities.txt]: These lines are dupplicates

Comment (realityforge) [reply java/elemental2/dom/integer_entities.txt]: They are different casing on `Screen` versus `screen`. FWIU they are both supported but `webkitRequestFullscreen` was preferred and it was also close to the standard complaint variant `requestFullscreen`

Comment (jDramaix) [reply java/elemental2/dom/integer_entities.txt]: You should maybe deprecate the `webkitRequestFullScreen` version

File: java/elemental2/dom/integer_entities.txt [hunk 2]
```diff
@@ -92,13 +92,16 @@ elemental2.dom.DOMTokenList.length
 elemental2.dom.DOMTokenList.item.index
 elemental2.dom.DOMImplementationList.length
 elemental2.dom.DOMImplementationList.item.index
+elemental2.dom.Element.ALLOW_KEYBOARD_INPUT
```
Comment (jDramaix) [inline java/elemental2/dom/integer_entities.txt]: It seems that Element.ALLOW_KEYBOARD_INPUT doesn't exist anymore? We should deprecate it or even remove it.
I'll check if it used internally and delete the api if not

Comment (realityforge) [reply java/elemental2/dom/integer_entities.txt]: Indeed it does look like it was deprecated and removed a while ago. I probably should have picked up in that. I was just poking around something else in that file and figured I would clean up as I went. :)

**Pull request closed without merging**